instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py

```diff
@@ -126,13 +127,15 @@ def process_request(self, request):
 
         token = attach(extract(request_meta, getter=wsgi_getter))
 
-        span = self._tracer.start_span(
-            self._get_span_name(request),
-            kind=SpanKind.SERVER,
+        server_span_tracer = ServerSpanTracer(
+            self._tracer, 
+            self._get_span_name(request), 
+            extract(request_meta, getter=wsgi_getter),
```

Comment:
Why only use `wsgi_getter` in all instrumentations? What if we are using asgi server? Would it make a difference?

Reply:
I'm not sure why wsgi_getter is used in all instrumentations, but it was used before I made my PR so I continued to use it when extracting context.